tests/test_houston_ingress.py

```diff
@@ -4,12 +4,44 @@
 from . import supported_k8s_versions
 
 
+def ingress_assertions_v1(doc):
+    assert doc["apiVersion"] == "networking.k8s.io/v1"
+    assert "RELEASE-NAME-houston" in [
+        name[0]
+        for name in jmespath.search(
+            "spec.rules[*].http.paths[*].backend.service.name", doc
+        )
+    ]
+    assert "houston-http" in [
+        port[0]
+        for port in jmespath.search(
+            "spec.rules[*].http.paths[*].backend.service.port.name", doc
+        )
+    ]
+
+
+def ingress_assertions_v1beta1(doc):
+    assert doc["apiVersion"] == "networking.k8s.io/v1beta1"
+    assert "RELEASE-NAME-houston" in [
+        name[0]
+        for name in jmespath.search(
+            "spec.rules[*].http.paths[*].backend.serviceName", doc
+        )
+    ]
+    assert "houston-http" in [
+        port[0]
+        for port in jmespath.search(
+            "spec.rules[*].http.paths[*].backend.servicePort", doc
+        )
+    ]
+
+
```

Comment:
@andriisoldatenko what do you think about this syntax? I'm not 100% sure that making functions like `ingress_assertions_v1` is a good idea. WDYT?

Reply:
in all places you have prefixed function with test_ , can we use the same here too

Reply:
If I prefix these functions with `test_` then pytest runs them as if they are individual test cases, and they are not.